tortoise/verifying_tortoise_test.go

```diff
@@ -3059,3 +3065,106 @@ func BenchmarkTortoiseLayerHandling(b *testing.B) {
 		}
 	}
 }
+
+func randomBytes(t *testing.T, size int) []byte {
+	data, err := crypto.GetRandomBytes(size)
+	require.NoError(t, err)
+	return data
+}
+
+func randomBlock(t *testing.T, lyrID types.LayerID, beacon []byte, refBlockID *types.BlockID) *types.Block {
```

Comment:
can you please use tb testing.TB and set tb.Helper(), to make it more reusable

Reply:
done